proto/buf.gen.yaml

```
@@ -0,0 +1,9 @@
+version: v1
+plugins:
+  - name: go
+    out: ..
+  - name: go-grpc
+    out: ..
+  - name: grpc-gateway
+    out: ..
+    opt: logtostderr=true
```

I think we can use `module` option.
And remove the script from `proto-gen` in `Makefile`.
```
        cp -R github.com/medibloc/panacea-oracle/pb ./
	rm -rf github.com
```

```suggestion
plugins:
  - name: go
    out: ..
    opt: module=github.com/medibloc/panacea-oracle
  - name: go-grpc
    out: ..
    opt: module=github.com/medibloc/panacea-oracle
  - name: grpc-gateway
    out: ..
    opt:
      - logtostderr=true
      - module=github.com/medibloc/panacea-oracle
```

---

Good point: 2f87b395e0ca01250a0585bc298dd20667779c93